collect_app/src/main/java/org/odk/collect/android/widgets/items/SelectOneFromMapDialogFragment.kt

```diff
@@ -106,14 +107,20 @@ internal class SelectChoicesMapData(private val resources: Resources, prompt: Fo
                 val latitude = geometry.split(" ")[0].toDouble()
                 val longitude = geometry.split(" ")[1].toDouble()
 
+                val properties = selectChoice.additionalChildren.filter {
```

Comment:
I think we'll likely want to consider doing this on opening of the bottom sheet rather than preemptively. I expect that would have an appreciable effect on performance.

Less significant, but this filter then map does two passes over the properties, right? I do expect there will be cases where the property list will be significant. Of course if we just do this on bottomsheet open it's not as important. Probably no action to take here but wanted to share that I had to think through this in case it sparks other ideas.